src/protocols/Http.ts

```diff
@@ -114,13 +152,13 @@ class HttpProtocol extends KuzzleAbstractProtocol {
         throw error;
       })
       .then(() => {
-        this._routes = Object.assign(this._routes, this.customRoutes);
+        this._routes = Object.assign(this._routes, this._customRoutes);
 
         // Client is ready
         this.clientConnected();
       })
       .catch(err => {
-        const connectionError = new Error(`Unable to connect to kuzzle server at ${this.host}:${this.port}`);
+        const connectionError: any = new Error(`Unable to connect to kuzzle server at ${this.host}:${this.port}`);
```

Comment:
Shouldn't this be...
```suggestion
        const connectionError: Error = new Error(`Unable to connect to kuzzle server at ${this.host}:${this.port}`);
```

Reply:
I can't use the `Error` type because the next line we add the original internal error in the `error.internal` property